### DISCUSSION TITLE
Addition to a calendar in the Agenda 44.1+ application

### DISCUSSION THREAD

**Discussion by HDInfautre** [alainm23/planify#1020] [Ideas]
hello

I'm using ubuntu 23.04 and I'm wondering if it would be possible to integrate the tasks of a project X into a project X calendar in the Agenda 44.1+ application?

_Thank you for this application, which I'm just discovering, as I have to give up my old task management application. It looks nice, ergonomic and functional.
I didn't see any "documentation" section, even if it's very intuitive, that would avoid asking you for help._

**Comment by alainm23** [comment]
I'm sorry but I didn't understand your request.

**Reply by HDInfautre** [reply]
Less important. some task managers add tasks to an agenda; mine is Ubuntu Agenda but perhaps too isolated case!?
